tests/pricing_models/test_get_max.py

```diff
@@ -28,7 +29,7 @@ class TestCaseGetMax:
 
 
 class TestGetMax(unittest.TestCase):
-    """Tests get_max_short and get_max_long functions within the pricing model."""
+    """Tests get_max_short and get_max_long functions within the pricing model"""
```

Comment:
we should pick some style standards for this, maybe in a follow-up PR we can do a mega-lint

Reply:
[ruff](https://beta.ruff.rs/docs/) would be great for this use-case:
```
Found 1852 errors.
[*] 1562 potentially fixable with the --fix option.
```

Reply:
Agreed! We should also fill in contributing.md